src/utils/errors.ts

```diff
@@ -5,6 +5,14 @@ import { FAILURE_MESSAGES } from '@graasp/translations';
 
 export const ConfigurationError = ErrorFactory('config');
 
+export abstract class CustomError extends Error {
```

Comment:
I feel like we could easily inherit CustomError on bad use cases. It shouldn't be returned to the client since there is no Error Code, then the name should probably be more system/backend related ?

Reply:
I'll rename it and move directly in the Repositories error.